test/net/sourceforge/kolmafia/persistence/HolidayDatabaseTest.java

```diff
@@ -19,15 +18,18 @@ class HolidayDatabaseTest {
   class HamburgularPosition {
     @Test
     void noPositionBeforeCollision() {
-      var date = ZonedDateTime.of(2006, Month.JUNE.getValue(), 2, 0, 0, 0, 0, ROLLOVER);
+      var date =
+          ZonedDateTime.of(2006, Month.JUNE.getValue(), 2, 0, 0, 0, 0, DateTimeManager.ROLLOVER);
```

Comment:
I think you could use a static import as before to save everything going across many lines ;)

Reply:
can!